Doc/source/designspaceLib/readme.rst

```diff
@@ -1076,19 +1074,20 @@ UFOs says.
 .. 8-implementation-and-differences:
 
 
-8 Implementation and differences
-================================
+Implementation and differences
+******************************
 
 The designspace format has gone through considerable development. 
 
  -  the format was originally written for MutatorMath.
- -  the format is now also used in fontTools.varlib.
+ -  the format is now also used in fontTools.designspaceLib.
  -  not all values are be required by all implementations.
 
-8.1 Varlib vs. MutatorMath
---------------------------
+8.1 designspaceLib vs. MutatorMath
```

Comment:
it's actually fontTools.varLib who consumes the designspace files. The designspaceLib module mostly only reads/write the raw data. Keep varLib in here